onnxruntime/core/providers/cpu/cpu_execution_provider.cc

```diff
@@ -66,6 +66,7 @@ class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOn
 class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 6, 12, uint64_t, Abs);
 class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 6, 12, float, Floor);
 class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 6, 12, float, Ceil);
+class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 6, 12, double, Ceil);
```

Comment:
How does versioning work? Is this line indicating that this operator is available in versions 6 - 12? Is this right? Should it just be available in future versions?

Reply:
public docs link to release branches of the OperatorKernels.md which list ORT operator support.
https://onnxruntime.ai/docs/reference/operators/OperatorKernels.html